packages/expo-blur/ios/BlurViewModule.swift

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022-present 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+
+public final class BlurViewModule: Module {
+  public func definition() -> ModuleDefinition {
+    Name("ExpoBlurView")
+
+    View(BlurView.self) {
+      Prop("tint") { (view, tint: String) in
```

Comment:
Maybe we could use the `UIColor` type here?

Reply:
Unfortunately not, the underlying tint must be an enum of type [`UIBlurEffect.Style`](https://developer.apple.com/documentation/uikit/uiblureffect/style). Making our own enum here would make more sense, but since there are a lot of unimplemented cases in that native enum, I'd like to work on it separately 😉